src/main/resources/db/migration/V24__newsletter_cultural_guides.sql

```
@@ -0,0 +1,20 @@
+CREATE TABLE IF NOT EXISTS newsletter_cultural_guides (
+                                                          id              BIGINT      GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+                                                          newsletter_id   BIGINT      NOT NULL,  -- FK → newsletter
+                                                          user_id         BIGINT      NOT NULL,  -- FK → users
+                                                          school_guide_id BIGINT      NOT NULL,  -- FK → school_guide
+                                                          display_order   INT         NOT NULL DEFAULT 0,  -- AI가 반환한 순서 (0부터)
+                                                          created_at      TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+                                                          updated_at      TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE INDEX IF NOT EXISTS idx_newsletter_cultural_guides_newsletter_id
+    ON newsletter_cultural_guides (newsletter_id);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)